Header header               #contains timestamp of most recent update to the object list
SegmentedObject[] objects   #list of segmented objects
